docs/integrations/sources/zendesk-support.md

```diff
@@ -51,16 +74,23 @@ The connector is restricted by normal Zendesk [requests limitation](https://deve
 The Zendesk connector should not run into Zendesk API limitations under normal usage. Please [create an issue](https://github.com/airbytehq/airbyte/issues) if you see any rate limit issues that are not automatically retried successfully.
 
 ## Getting started
-
+## CHANGELOG
+| Version | Date | Pull Request | Subject |
+| :------ | :--------  | :-----       | :------ |
+| `0.1.0` | 2021-07-21 | [4861](https://github.com/airbytehq/airbyte/issues/3698) | created CDK native zendesk connector |
```

Comment:
`CHANGELOG` should probably be moved to the end of this doc. The `Requirements` and `Setup guide` below do not belong to `CHANGELOG`.

Reply:
Done